project/ws/app/src/lib/head/left-menu/left-menu.component.scss

```diff
@@ -110,9 +116,10 @@
 }
 
 ::ng-deep .menuPannel .mat-expansion-panel-header {
-  padding: 8px 16px !important;
```

Comment:
same as above

Reply:
Okay will do it

Reply:
menuPannel is the parent class have checked